packages/swap/src/event-handlers/ccmDepositReceived.ts

```diff
@@ -1,38 +0,0 @@
-import { z } from 'zod';
-import { hexString, u128, u64 } from '@/shared/parsers';
-import { encodedAddress } from './common';
-import type { EventHandlerArgs } from '.';
-
-const ccmDepositReceivedArgs = z.object({
-  ccmId: u64,
-  principalSwapId: u64.nullable().optional(),
-  gasSwapId: u64.nullable().optional(),
-  depositAmount: u128,
-  destinationAddress: encodedAddress,
-  depositMetadata: z.object({
-    channelMetadata: z.object({
-      message: hexString,
-      gasBudget: u128,
-    }),
-  }),
-});
-
-export type CcmDepositReceivedArgs = z.input<typeof ccmDepositReceivedArgs>;
-
-/** @deprecated remove after 1.6 is on mainnet */
-export default async function ccmDepositReceived({ prisma, event, block }: EventHandlerArgs) {
-  const { principalSwapId, depositMetadata } = ccmDepositReceivedArgs.parse(event.args);
-
-  if (principalSwapId) {
-    await prisma.swapRequest.update({
-      where: {
-        nativeId: principalSwapId,
-      },
-      data: {
-        ccmDepositReceivedBlockIndex: `${block.height}-${event.indexInBlock}`,
```

Comment:
can we remove this field? thinks its not used anymore

Reply:
It's used in the status endpoint, we'd have to deprecated it first but we are already deprecating that endpoint anyway